using_images/sti_images/php.adoc

```diff
@@ -26,58 +33,97 @@ This image comes in two flavors, depending on your needs:
 
 The RHEL 7 image is available through Red Hat's subscription registry via:
 
-****
-`$ docker pull registry.access.redhat.com/openshift/php-55-rhel7`
-****
+----
+$ docker pull registry.access.redhat.com/openshift/php-55-rhel7
+----
 
 *CentOS 7 Based Image*
 
 This image is available on DockerHub. To download it:
 
-****
-`$ docker pull openshift/php-55-centos7`
-****
+----
+$ docker pull openshift/php-55-centos7
+----
 
-To use these images, you can either access them directly from those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
+To use these images, you can either access them directly from these
+registries, or push them into your OpenShift docker registry. Additionally,
```

Comment:
no comma needed after registries.

no comma needed after "to use these images" either i don't think.